install_files/ansible-base/roles/restore/tasks/main.yml

```diff
@@ -1,22 +1,81 @@
 ---
-- name: Copy restore script to Application Server.
-  copy:
-    src: restore.py
-    dest: /tmp/
-    owner: root
-    mode: "0770"
-
-- name: Copy backup tarball of config info to Application Server.
-  copy:
+- name: Create temporary directory for Tor configuration check
+  connection: local
+  become: no
+  tempfile:
+    state: directory
+  register: torrc_check_dir
+
+- name: Fetch current Tor configuration from app server
+  become: no
+  fetch:
+    src: /etc/tor/torrc
+    dest: "{{ torrc_check_dir.path }}"
```

Comment:
First thought was that we needn't copy the `torrc` back to the Admin Workstation, but doing so allows us to compare the files _before_ we copy the potentially-quite-large tarball to the remote. Great choice. :+1: